phpunit.xml.dist

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/5.6/phpunit.xsd"
+         backupGlobals="false"
+         colors="true"
+         bootstrap="app/autoload.php">
+    <testsuites>
+        <testsuite name="Sylius Standard Test Suite">
+            <directory>tests</directory>
+        </testsuite>
+    </testsuites>
+
+    <php>
+        <server name="KERNEL_CLASS_PATH" value="/app/TestAppKernel.php" />
+        <server name="IS_DOCTRINE_ORM_SUPPORTED" value="true" />
```

Comment:
It's specific to Lakion/ApiTestCase and I guess it is still required. /cc @lchrusciel